scale/job/tasks/manager.py

```diff
@@ -20,6 +22,50 @@ def __init__(self):
         self._tasks = {}  # {Task ID: Task}
         self._lock = threading.Lock()
 
+    def generate_status_json(self, nodes_list):
```

Comment:
Didn't see any unit test coverage for this function.

Reply:
Added unit test